services/transcode/merge.go

```diff
@@ -172,7 +173,8 @@ func MergeSubtitles(input common.MergeInput, progressCallback ffmpeg.ProgressCal
 	// for each file, extract the specified range and save the result to a file.
 	for index, item := range input.Items {
 		file := filepath.Join(input.WorkDir, fmt.Sprintf("%d.srt", index))
-		cmd := exec.Command("ffmpeg", "-i", item.Path, "-ss", fmt.Sprintf("%f", item.Start), "-to", fmt.Sprintf("%f", item.End), "-y", file)
+		path := utils.IsilonPathFix(item.Path)
+		cmd := exec.Command("ffmpeg", "-i", path, "-ss", fmt.Sprintf("%f", item.Start), "-to", fmt.Sprintf("%f", item.End), "-y", file)
```

Comment:
```suggestion
		cmd := exec.Command(
		"ffmpeg", 
		"-i", path, 
		"-ss", fmt.Sprintf("%f", item.Start), 
		"-to", fmt.Sprintf("%f", item.End), 
		"-y", file,
		)
```

Makes it significantly easier to read the command IMO